.gitignore

```diff
@@ -33,3 +33,4 @@
 yarn-debug.log*
 .yarn-integrity
 .DS_Store
+.env
```

Comment:
Let's add `.env.example` file at root level of the project. which will have all env variable but with dummy/fake values and will not be added in `.gitignore`